src/components/app-core/typography.css

```diff
@@ -1,11 +1,3 @@
-.hero {
```

Comment:
Just deleting some styles that aren't used, could be on a different PR if you want.